relay_config/config.yml

```
@@ -0,0 +1,16 @@
+---
+relay:
+  upstream: "http://web:9000/"
+  host: 0.0.0.0
+  port: 3000
+logging:
+  level: TRACE
```

```suggestion
  level: WARN
```

Let's not trace as it would be too noisy and consume a lot of disk space.

---

no logging configuration by default, 
leave only a commented out configuration so the user can easily change it to whatever.